src/notebook.cpp

```diff
@@ -122,6 +122,13 @@ class mKCal::Notebook::Private
     QString mSyncProfile;
     QDateTime mCreationDate;
     QHash<QByteArray, QString> mCustomProperties;
+
+    void setModified()
```

Comment:
Maybe something like updateModified() to separate it from setModifiedDate()?

Reply:
Good idea. Changed for `updateModified`.